thirdparty/gflags/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+PROJECT(gflags)
+
+set(TARGET_NAME gflags)
+
```

Comment:
I suppose we don't need `project` and target name, while we need to add `add_subdirectory`